scripts/app.js

```diff
@@ -138,6 +138,12 @@ angular.module('trackerCapture')
     $http.get('components/dataentry/section-inner-form.html').then(function(page){
         $templateCache.put('components/dataentry/section-inner-form.html', page.data);
     });
+    $http.get('views/dialog.html').then(function(page){

```

Comment:
Is this template cache actually used? It was hard to see where the dialog.html and modal-body.html is retrieved from cache...